EFCore.NamingConventions/NamingConventions/Internal/NamingConvention.cs

```diff
@@ -3,6 +3,7 @@ namespace EFCore.NamingConventions.Internal
     enum NamingConvention
     {
         None,
-        SnakeCase
+        SnakeCase,
+        AllLowerCase,
```

Comment:
Let's rename to LowerCase to keep things consistent with the name of the convention